contracts/test/v0.8/foundry/vrf/VRFV2Plus.t.sol

```diff
@@ -452,4 +459,8 @@ contract VRFV2Plus is BaseTest {
     (uint96 linkBalanceAfter, , , , ) = s_testCoordinator.getSubscription(subId);
     assertApproxEqAbs(linkBalanceAfter, linkBalanceBefore - 280_000, 20_000);
   }
+
+  function testMyFunction() public {
```

Comment:
*nit. remove?

Reply:
Yikes, good find. Committed this by mistake.